### ISSUE TITLE
Add a11y

### ISSUE THREAD

**Issue by sneakers-the-rat** [labki-org/LabkiPageFormsInputs#4]
looks like flatpickr is stalled out on accessibility - https://github.com/flatpickr/flatpickr/discussions/2684

but it is pretty easy to do.

p much every input should have accessibility properties, and date pickers in particular can be bewildering on a screenreader. making the picker be accessible would require upstream changes probably, but since the input is ultimately a textbox here, we can at least make sure it's annotated properly. 

w3c has some examples e.g. here: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-datepicker/
